mis_builder_budget/models/mis_budget_abstract.py

```diff
@@ -33,7 +33,7 @@ def copy(self, default=None):
             default = {}
         if "name" not in default:
             default["name"] = _("%s (copy)") % self.name
-        return super(MisBudgetAbstract, self).copy(default=default)
+        return super().copy(default=default)
```

Comment:
Can you do these changes that are not related to migration in a separate commit (or better, a separate PR), so they can be backported easily ?